frame/collective/src/lib.rs

```diff
@@ -146,16 +153,16 @@ impl<AccountId, I> GetBacking for RawOrigin<AccountId, I> {
 }
 
 /// Info for keeping track of a motion being voted on.
-#[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
-pub struct Votes<AccountId, BlockNumber> {
+#[derive(PartialEq, Eq, Clone, Encode, Decode, TypeInfo, MaxEncodedLen, RuntimeDebug)]
+pub struct Votes<BoundedAccountIdVec, BlockNumber> {
 	/// The proposal's unique index.
 	index: ProposalIndex,
 	/// The number of approval votes that are needed to pass the motion.
 	threshold: MemberCount,
 	/// The current set of voters that approved it.
-	ayes: Vec<AccountId>,
+	ayes: BoundedAccountIdVec,
```

Comment:
Wouldn't it make more sense to have this type instead?

```suggestion
	ayes: BoundedVec<AccountId, MaxMembers>,
```

Reply:
@KiChjang yeah, but the derive macros start freaking out because of `MaxMembers`. Do me a favor and try it out, and if you can make it work, feel free to commit :)

Reply:
25aaf6ea5f99db7e3fc4a1570aa90a448fdee83a here you go :)